### ISSUE TITLE
Test cases regarding NickServ identification

### ISSUE THREAD

**Issue by michalrud** [RITlug/teleirc#76]
There are some test cases that would be nice to write for #65 , unfortunately I don't have time to do so in a timely manner :(

1. Test checking that joining the channel doesn't happen before nickserv answers
2. Test checking that receiving a message from anyone else than nickserv doesn't trigger joining the channel
3. Consider parsing NickServ messages to understand if we're really identified

If there are any questions, feel free to contact me :) There are also some tests in `IrcConnectionTests.js` on which new test can base on.

**Comment by jwflory** [comment]
Since we are starting from scratch for the Go v2.x.x rewrite (#163), this will fit into our existing discussions about writing tests (#236). Closing this issue as `wontfix` since we are not planning to support the v1.x.x versions once the rewrite is complete.
